econplayground/urls.py

```diff
@@ -41,6 +41,8 @@
          views.GraphDetailView.as_view(), name='cohort_graph_detail'),
     path('course/<int:cohort_pk>/graph/<int:pk>/edit/',
          views.FeaturedGraphUpdateView.as_view(), name='cohort_graph_edit'),
+    path('course/<int:cohort_pk>/graph/<int:pk>/clone/',
```

Comment:
Is this hooked up in the interface anywhere?

Reply:
^ nm. Just saw the other PR.

Reply:
It is here: https://github.com/ccnmtl/econplayground.js/pull/560

Or for testing purposes you can just go there directly.